docker/env/version

```diff
@@ -1 +1 @@
-1.38-argus-0.10.2-update
+1.38-argus-0.11.1-update
```

Comment:
Update the number at the beginning, the fact argurs version number is part of this name is just cause the last update was related to argus, and we are lazy to rebuild the docker image before merges.